x/auction/types/codec.go

```diff
@@ -35,8 +34,4 @@ func init() {
 	RegisterLegacyAminoCodec(amino)
 	cryptocodec.RegisterCrypto(amino)
 	sdk.RegisterLegacyAminoCodec(amino)
-
-	// Register all Amino interfaces and concrete types on the authz and gov Amino codec so that this can later be
-	// used to properly serialize MsgSubmitProposal instances
-	RegisterLegacyAminoCodec(govcodec.Amino)
```

Comment:
why remove this? do we not have any gov txs for auction?

Reply:
`govcodec.Amino` isn't available anymore, there's no way to do it in sdk 50 afaict. i have a personal todo to check this in integration tests (i think we already have a test for this there).